content/user/ui/settings/Roles.md

```diff
@@ -5,32 +5,34 @@ tags: "TeamServer roles observer superadmin system"
 
 ## Organization Roles
-Organization roles have always existed in TeamServer. Every user is provided a default role for the default organization. Users that are included in organization access control groups may be granted various roles across different organizations based on their involvment with a specific organization and the applications associated with that organization. There are four primary organization roles. In addition, a fifth role called "no access" can be used to purposely exclude one or more users from accessing the organization.
+Every user is provided a default role for the default organization. Users that are included in organization access control groups may be granted various roles across different organizations based on their involvement with a specific organization and the applications associated with that organization. There are four primary organization roles and a fifth role used to exclude one or more users from accessing the organization.
 
 | Role                 | Description                                                                                                                                                                                                                                                          |
 |----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Administrator | The organization administrator is responsible for configuration of the organization.This user has the ability license applications, manage user accounts and perform other advanced configuration functions found in the organization [settings menu](user_tsguideui.html#overview). |
-| Editor        | The organization editor has the ability to interact with the full lifecycle of traces and vulnerabilities. This particular user has no administration capabilities. This user can bring applications online. He/she can delete them as well. No organization administration functions can be performed by a user with this role. The vast majority of users of TeamServer will have this role. |
-| Rules Admin   | This user has edit privilege to rules. Essentially, this user role inherits all of the same functionality as a user with the editor role. |
-| View          | Some organizations want their users to have read-only access to the findings presented in TeamServer. This role was designed to accommodate read-only access. |
+| Administrator | The Organization Administrator is responsible for configuration of the organization. This user has the ability to license applications, manage user accounts and perform other advanced configuration functions found in the organization [settings menu](user_tsguideui.html#overview). |
```

Comment:
Thinking we maybe change the link text here to say "Organization Settings"? So, "...found in Organization Settings."